_templates/usecase/new/command.ejs.t

```diff
@@ -1,13 +1,8 @@
 ---
 to: apps/api/src/app/<%= module %>/usecases/<%= name %>/<%= name %>.command.ts
 ---
-import { CommandHelper } from '../../../shared/commands/command.helper';
 import { EnvironmentWithUserCommand } from '../../../shared/commands/project.command';
 
-export class <%= h.changeCase.pascal(name) %>Command extends EnvironmentWithUserCommand {
-  static create(data: <%= h.changeCase.pascal(name) %>Command) {
-    return CommandHelper.create<<%= h.changeCase.pascal(name) %>Command>(<%= h.changeCase.pascal(name) %>Command, data);
-  }
-}
+export class <%= h.changeCase.pascal(name) %>Command extends EnvironmentWithUserCommand {}
```

Comment:
I've found this template and removed the `create` function. The `EnvironmentWithUserCommand` extends `BaseCommand` that has this static function defined

Reply:
That's great! Will help to make sure we are consistent with this for new usecases too!